packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -249,6 +249,12 @@ export class PendingStateManager implements IDisposable {
 			return;
 		}
 
+		if (pendingMessage.messageType === ContainerMessageType.IdAllocation) {
+			// The stashedState isn't sent over the wire so comparison of message
```

Comment:
Where does this comparison happen?

Reply:
Comparison happens here: https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/pendingStateManager.ts#L256